test/membrane_portaudio_plugin/source_test.exs

```diff
@@ -68,7 +71,7 @@ defmodule Membrane.Portaudio.SourceTest do
     test "after starting some buffers should be received", %{state: state} do
       {:ok, resource_guard} = Membrane.Testing.MockResourceGuard.start_link()
 
-      assert {{:ok, [_actions]}, _state} =
+      assert {[_actions], _state} =
```

Comment:
Shouldn't it be `_action`, since we expect only a single element list of actions to be returned? :D